messaging/3.1/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Jakarta Messaging Spec 3.1 (Under Development)"
```

Comment:
@OndroMih Looks good! Can you remove "Spec" from the title so it is conform with the previous versions?

Reply:
Done.